File: 240413/없는 번호 찾아내기/find-missing-number.kt
fun main() {
    val num = IntArray(30) { 0 }

    for( i in 0 until 30) {
        readLine()?.toIntOrNull()?.let {
            num[i] = it
        }
    }

    val temp = IntArray(30) { 0 }
    for( j in 0 until 30) {
        for( k in 1..30) {
            if(num[j] == k) {
                temp[k-1] = num[j]
            }
        }
    }

    for( m in 0 until 30) {
        if(temp[m] == 0) {
            println(m+1)
        }
    }
}

File: 240413/없는 번호 찾아내기/find-missing-number.c
#include <stdio.h>

int main() {
    // 여기에 코드를 작성해주세요.
    int num[30] = { 0 };
    int i;

    for(i=0; i<30; i++) {
        scanf("%d", &num[i]);
    }

    int j, k, temp[30] = { 0 };
    for(j=0; j<30; j++) {
        for(k=1; k<=30; k++) {
            if(num[j] == k) {
                temp[k-1] = num[j];
            }
            else
                continue;
        }
    }

    int ans[2] = { 0 }, m;
    for(m=0; m<30; m++) {
        if(temp[m] == 0) 
            printf("%d\n", m+1);
    }
    
    return 0;
}